js/main.js

```diff
@@ -3,20 +3,27 @@
 
 Site = {
   mobileThreshold: 601,
+  $body: $('body'),
+  $mailchimpForm: $('#mailchimp-form'),
```

Comment:
ok cool. so init mailchimp in Site.init() and do conditional in Site.Mailchimp.init() ?